test/tests/setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+
+yum -y -q install yum-plugin-copr
+yum -y -q copr enable lmr/Autotest
+yum -y -q install avocado
+yum -y -q install avocado-virt
+
+# because of troubles with cockpit recompiled to /opt/ instead of /usr
+#setenforce 0
+source lib/create_test_machine.sh
+
+PREFIX=checkmachine2
+DISTRO=fedora-21
+virt-create $PREFIX $DISTRO 
+NAME=`vm_get_name $PREFIX $DISTRO`
+IP=`vm_get_ip $NAME`
+
+/bin/cp -r * /usr/share/avocado/tests
```

Comment:
Hi,
This is default localtion, where avocado is searching for tests. It will be used in case of avocado virt plugin. Now it is not needed to have it here.
But finally it will be only one location of tests, and test will be copied only there, not to virt machine directly.